src/controllers/comment.controller.js

```diff
@@ -24,6 +25,29 @@ class CommentController {
     ResponseService.setSuccess(201, 'Your comment was submitted successfully', commentInformation);
     return ResponseService.send(res);
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @returns {object} This function returns list of comments posted on thread
+   * @memberof CommentController
+   */
+  static async viewCommentPostedOnTripRequest(req, res) {
```

Comment:
@erickyvand If you think about this, we might not necessarily need a comment controller. Instead, we need a comment service that will be used by both the tripController and accommodationController to manage comments/feedback.

Reply:
On this feedback, after explaining to me why we don't need a comment controller, you told me that we can proceed like the way it is.